app/views/layouts/application.html.haml

```diff
@@ -3,7 +3,8 @@
 - content_for :page_title, "#{@page_title.present? ? "#{@page_title} - GOV.UK Registers" : 'GOV.UK Registers' }"
 
 - content_for :after_header do
-  %header{class: "header #{current_page?(root_path) ? 'header--without-border' : 'header'}"}
+  -# %header{class: "header #{current_page?(root_path) ? 'header--without-border' : 'header'}"}
```

Comment:
Should we add a `TODO` on the comment to restore so it is more easily identified?